### ISSUE TITLE
"service unavailable" result of translation

### ISSUE THREAD

**Issue by farmazon3000** [bpierre/gtranslate#43]
it happens again, results of ```about:support```

```
{
  "application": {
    "name": "Firefox",
    "version": "36.0.1",
    "buildID": "20150305021524",
    "userAgent": "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:36.0) Gecko/20100101 Firefox/36.0",
    "updateChannel": "release",
    "supportURL": "https://support.mozilla.org/1/firefox/36.0.1/WINNT/en-US/",
    "numTotalWindows": 1,
    "numRemoteWindows": 0
  },
  "crashes": {
    "submitted": [],
    "pending": 0
  },
  "modifiedPreferences": {
    "accessibility.typeaheadfind.flashBar": 0,
    "browser.cache.disk.smart_size.first_run": false,
    "browser.cache.disk.capacity": 358400,
    "browser.cache.frecency_experiment": 1,
    "browser.places.smartBookmarksVersion": 7,
    "browser.sessionstore.upgradeBackup.latestBuildID": "20150305021524",
    "browser.startup.homepage_override.mstone": "ignore",
    "browser.startup.homepage_override.buildID": "20150305021524",
    "dom.mozApps.used": true,
    "extensions.lastAppVersion": "36.0.1",
    "gfx.direct3d.last_used_feature_level_idx": 0,
    "media.gmp-gmpopenh264.lastUpdate": 1434361047,
    "media.gmp-gmpopenh264.version": "1.3",
    "media.gmp-manager.lastCheck": 1435566307,
    "network.cookie.prefsMigrated": true,
    "places.database.lastMaintenance": 1435484644,
    "places.history.expiration.transient_current_max_pages": 104858,
    "plugin.importedState": true,
    "plugin.disable_full_page_plugin_for_types": "application/pdf",
    "privacy.sanitize.migrateFx3Prefs": true,
    "storage.vacuum.last.places.sqlite": 1434361215,
    "storage.vacuum.last.index": 1
  },
  "lockedPreferences": {},
  "graphics": {
    "numTotalWindows": 1,
    "numAcceleratedWindows": 1,
    "windowLayerManagerType": "Direct3D 11",
    "windowLayerManagerRemote": true,
    "adapterDescription": "NVIDIA GeForce 210 ",
    "adapterVendorID": "0x10de",
    "adapterDeviceID": "0x0a65",
    "adapterSubsysID": "35301458",
    "adapterRAM": "1024",
    "adapterDrivers": "nvd3dumx,nvwgf2umx,nvwgf2umx nvd3dum,nvwgf2um,nvwgf2um",
    "driverVersion": "9.18.13.1407",
    "driverDate": "2-9-2013",
    "adapterDescription2": "",
    "adapterVendorID2": "",
    "adapterDeviceID2": "",
    "adapterSubsysID2": "",
    "adapterRAM2": "",
    "adapterDrivers2": "",
    "driverVersion2": "",
    "driverDate2": "",
    "isGPU2Active": false,
    "direct2DEnabled": true,
    "directWriteEnabled": true,
    "directWriteVersion": "6.2.9200.16571",
    "webglRenderer": "Google Inc. -- ANGLE (NVIDIA GeForce 210  Direct3D9Ex vs_3_0 ps_3_0)",
    "info": {
      "AzureCanvasBackend": "direct2d",
      "AzureSkiaAccelerated": 0,
      "AzureFallbackCanvasBackend": "cairo",
      "AzureContentBackend": "direct2d"
    }
  },
  "javaScript": {
    "incrementalGCEnabled": true
  },
  "accessibility": {
    "isActive": false,
    "forceDisabled": 0
  },
  "libraryVersions": {
    "NSPR": {
      "minVersion": "4.10.7",
      "version": "4.10.7"
    },
    "NSS": {
      "minVersion": "3.17.4 Basic ECC",
      "version": "3.17.4 Basic ECC"
    },
    "NSSUTIL": {
      "minVersion": "3.17.4",
      "version": "3.17.4"
    },
    "NSSSSL": {
      "minVersion": "3.17.4 Basic ECC",
      "version": "3.17.4 Basic ECC"
    },
    "NSSSMIME": {
      "minVersion": "3.17.4 Basic ECC",
      "version": "3.17.4 Basic ECC"
    }
  },
  "userJS": {
    "exists": false
  },
  "extensions": [
    {
      "name": "gTranslate",
      "version": "0.9.1-signed",
      "isActive": true,
      "id": "{aff87fa2-a58e-4edd-b852-0a20203c1e17}"
    }
  ],
  "experiments": []
}
```

**Comment by bpierre** [comment]
> "version": "0.9.1-signed",

0.9.1 is an old version, please install the latest version of gtranslate (0.11.0.1-signed).

https://addons.mozilla.org/firefox/addon/gtranslate/

**Comment by CatQuest** [comment]
hi, no matter how much I got to https://addons.mozilla.org/en-US/firefox/addon/gtranslate/ and try re downloading to (0.11.0.1-signed). I always get 0.9.1 instead. and it requires restart as well. I did some digging and it turns out that it's because 11 doesn't support FF 24?
is it possible to find a workaround for this?

**Comment by farmazon3000** [comment]
"no matter how much I got to https://addons.mozilla.org/en-US/firefox/addon/gtranslate/ and try re downloading to (0.11.0.1-signed). I always get 0.9.1 instead."
I had the same so I've updated firefox and now I can install 0.11.

**Comment by mote0230** [comment]
Back in the day I fixed the 0.9.1
Find it here http://a.1339.cf/bvfssk.xpi
There's still some bugs now and then where it displays too much stuff but it works
